### PR TITLE
chore(client): rich print null when table have too many cols

### PR DESCRIPTION
**Pull request by goldenxinxing**
## Description
image

## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/core/eval/cli.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from starwhale.consts.env import SWEnv
 
 from .view import JobTermView, get_term_view, DEFAULT_PAGE_IDX, DEFAULT_PAGE_SIZE
+from ...consts import DEFAULT_REPORT_COLS
```
Comment (tianweidut) [inline client/starwhale/core/eval/cli.py:9]: `from starwhale.consts import DEFAULT_REPORT_COLS` ?

**Comment by tianweidut** [comment]
Others LGTM

@goldenxinxing  add some ut?

File: client/starwhale/core/eval/cli.py (hunk 1 above)
Comment (goldenxinxing) [reply client/starwhale/core/eval/cli.py]: fixed

**Comment by goldenxinxing** [comment]
> Others LGTM
> 
> @goldenxinxing add some ut?

done

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**